crates/storage-api/src/invocation_status_table/mod.rs

```diff
@@ -180,8 +183,38 @@ impl JournalMetadata {
     }
 }
 
+/// This is similar to [ServiceInvocation], but allows many response sinks,
+/// plus holds some inbox metadata.
+#[derive(Debug, Clone, PartialEq)]
+pub struct InboxedInvocationMetadata {
```

Comment:
Maybe just calling it `InboxedInvocation` since it is not only invocation metadata?